tests/functional/test_commands.py

```diff
@@ -65,6 +65,13 @@ def tearDown(self):
                 if exc.errno != errno.ENOENT:
                     raise
 
+    def file_contains(self, filename, text):
+        with open(filename,'r') as f:
+            output = f.read()
+            return (text in str(output))
```

Comment:
I think `output` will always be a string here already, and you can leave out the outer parentheses. 

```python
return text in output
```